pages/linux/xbps.md

```diff
@@ -0,0 +1,23 @@
+# xbps
+
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+
+- Install packages:
+
+`xbps-install -S package_name1 package_name2`
+
+- Search a package:
```

Comment:
Would `Search for a package` read better?

Reply:
yep, it sounds way better, thanks